companion/src/constants.h

```diff
@@ -59,10 +59,10 @@ enum BoardEnum {
 #define CPN_MAX_KEYS                   32
 #define CPN_MAX_MOUSE_ANALOGS          2
 
-const char * const ARROW_LEFT = "\xE2\x86\x90";
-const char * const ARROW_UP = "\xE2\x86\x91";
-const char * const ARROW_RIGHT = "\xE2\x86\x92";
-const char * const ARROW_DOWN = "\xE2\x86\x93";
+const char * const ARROW_LEFT = "\u2190";
+const char * const ARROW_UP = "\u2191";
+const char * const ARROW_RIGHT = "\u2192";
+const char * const ARROW_DOWN = "\u2193";
```

Comment:
This produces a mountain of warnings on VC... French system language might have to do with it

`8>C:\opentx-22\code\companion\src\constants.h(64): warning C4566: character represented by universal-character-name '\u2192' cannot be represented in the current code page (1252) (compiling source file C:\opentx-22\code\companion\src\simulation\simulatorinterface.cpp)`